step-automation-packages/step-automation-packages-junit/src/test/java/step/client/StepRunnerYamlPlanTest.java

```diff
@@ -19,11 +19,16 @@
 package step.client;
 
 import org.junit.runner.RunWith;
+import step.automation.packages.junit.AutomationPackagePlans;
 import step.handlers.javahandler.AbstractKeyword;
 import step.junit.runner.Step;
+import step.junit.runners.annotations.ExecutionParameters;
 import step.junit.runners.annotations.Plans;
 
 @RunWith(Step.class)
-@Plans({"composite-simple-plan.yml"})
+@Plans({"plans/composite-simple-plan.yml"})
+@AutomationPackagePlans(value = {"plans/composite-simple-plan.yml"})
 public class StepRunnerYamlPlanTest extends AbstractKeyword {
```

Comment:
@david-stephan 
I noticed that the AP is read twice during local execution
* 1st time from `AutomationPackageKeywordsPlugin`
* 2nd time in `Step` runner

As I remember, the `AutomationPackageKeywordsPlugin` was initially designed to support use of keywords defined in current AP if we simply run a plan-text plan without AP junit runner. 

But after your last changes the old `Step` runner supports APs, so the `AutomationPackageKeywordsPlugin` probably can be deleted.